projects/listening-comp/services/transcript-processor/app.py

```
@@ -121,39 +165,73 @@ def health_check():
+        # Validate metadata
+        metadata_error = guardrails.validate_video_metadata(metadata)
+        if metadata_error:
+            return jsonify({'error': metadata_error}), 400
+
+        # Try to get transcript with multiple language codes and auto-generated option
+        for lang_code in ['ko', 'ko-KR']:
+            try:
+                # Try manual subtitles first
+                transcript = YouTubeTranscriptApi.get_transcript(video_id, languages=[lang_code])
+                break
+            except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)